flips/20201125-bip-44-multi-account.md

```diff
@@ -5,192 +5,282 @@
 | **FLIP #**    | [200](https://github.com/onflow/flow/pull/200) |
 | **Author(s)** | Peter Siemens (peter@dapperlabs.com), Jeffery Doyle (jeffrey.doyle@dapperlabs.com) |
 | **Sponsor**   | Peter Siemens (peter@dapperlabs.com) |
-| **Updated**   | 2020-11-25 |
+| **Updated**   | 2021-12-14 |
 
 ## Objective
 
-This document aims to define a standard pattern for the application of 
-[Bitcoin Improvement Proposal 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) (BIP 44)
-in Flow-compatible hardware and software wallets.
+This document aims to define a standard pattern for the application of [Bitcoin
+Improvement Proposal
+44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) (BIP 44) in
+Flow-compatible hardware and software wallets.
 
 ## Motivation
 
 This proposal was prompted by the recent development of the first Flow hardware
-wallet application (Ledger) and its companion host software (Flow Port). 
+wallet application (Ledger) and its companion host software (FCL-Ledger-Web among
+others).
 
-In the near term, this proposal affects users of the Flow Ledger application.
-In the long term, it affects Flow wallet developers and their users.
+In the near term, this proposal affects users of the Flow Ledger application. In
+the long term, it affects Flow wallet developers and their users.
 
 ## User Benefit
 
-By following this standard, developers of existing BIP 44 crypto wallets will
-be able to reduce friction for users who wish to create accounts on Flow.
+By following this standard, developers of existing BIP 44 crypto wallets will be
+able to reduce friction for users who wish to create accounts on Flow.
+
+By standardizing BIP 44 usage across wallets, users should be able to use their
+BIP 39 mnemonic codes across wallets to manage access to the same keys and accounts.
 
 ## Design Proposal
 
 ### Path Levels
 
-BIP 44 defines an implementation of 
-[Bitcoin Improvement Proposal 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
-that standardizes a format for hierarchical deterministic (HD) 
-key derivation paths that support multiple coins and multiple accounts per coin.
+BIP 44 defines an implementation of [Bitcoin Improvement Proposal
+32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
```

Comment:
The issue with BIP44 and BIP32 is that they only define the derivation on curve secp256k1. 
Referencing BIP32 shouldn't be enough to apply the derivation to another curve, unless we define how we want to generalize the BIP32 technique to other curves. 

There is a [SLIP](https://github.com/satoshilabs/slips/blob/master/slip-0010.md) that did something similar  "SLIP-0010 describes how to derive private and public key pairs for curve types different from secp256k1." but I'm having minor concerns with it. 
 I'll reach out to Vacuum Labs and maybe Ledger to see if there is a known generalization we can reference, otherwise we might need to describe the generalization ourselves (not long). I'll follow up on this.

Reply:
Update: I reached out to Ledger, but I looks like the answer will come after the holidays. Their answer should help us update FLIP200 (the current doc cannot be complete without specifying this curve derivation point unfortunately) 

cc @JeffreyDoyle

Reply:
Thanks @tarakby. Let's move this discussion to #200 so it doesn't get lost